checkov/cloudformation/parser/cfn_yaml.py

```diff
@@ -251,4 +256,15 @@ def load(filename: Path, content_type: ContentType) -> Tuple[DictNode, List[Tupl
 
     file_lines = [(idx + 1, line) for idx, line in enumerate(content.splitlines(keepends=True))]
 
+    if content_type == ContentType.TFPLAN:
```

Comment:
@gruebel Why just for tf plans?

Reply:
because this is actually the YAML parser, which parses JSONs better than our JSON parser. But I will check for the file type here, then also others can benefit, if they are JSON files.

Reply:
I meant, why not do it always?

Reply:
you can't parse YAML files with the json parser, it will result an exception. I adjusted it to do the fallback, if it is a JSON file now 🙂